recipes/spades/meta.yaml

```diff
@@ -1,29 +1,49 @@
+{% set version = "3.11.0" %}
+
 package:
   name: spades
-  version: '3.10.1'
+  version: {{ version }}
 
 source:
-  fn: SPAdes-3.10.1-Linux.tar.gz                                              # [linux]
-  url: http://spades.bioinf.spbau.ru/release3.10.1/SPAdes-3.10.1-Linux.tar.gz  # [linux]
-  md5: fb58d48a0545211aab91f43b7d17ad1b # [linux]
-  fn: SPAdes-3.10.1-Darwin.tar.gz                                             # [osx]
-  url: http://spades.bioinf.spbau.ru/release3.10.1/SPAdes-3.10.1-Darwin.tar.gz # [osx]
-  md5: f23bbad7921cff2c283e6589ccee83a2 # [osx]
+  fn: SPAdes-{{ version }}.tar.gz
+  url: http://cab.spbu.ru/files/release{{ version }}/SPAdes-{{ version }}.tar.gz
+  sha256: 308aa3e6c5fb00221a311a8d32c5e8030990356ae03002351eac10abb66bad1f
 
 build:
   number: 0
-  skip: True  # [not py2k and not py35]
 
 requirements:
   build:
+    - gcc >=4.8.2  # [not osx]
+    - llvm  # [osx]
+    - cmake >=2.8.12
+    - zlib
+    - bzip2
```

Comment:
This one as well please: https://github.com/conda-forge/conda-forge.github.io/blob/0661e2ecf4b213ddf36350ca100a5a820eb17eb7/scripts/pin_the_slow_way.py#L40-L85

Reply:
in most bioconda-recipes/recipes/*/meta.yaml zlib is not pinned.  same for bzip2.  is that a bug?